tests/test_source.py

```diff
@@ -66,7 +66,9 @@ def test_THREDDSMergedSource(THREDDSMergedSource_cat):
 def test_THREDDSMergedSource_long_short(THREDDSMergedSource_cat, THREDDSMergedSource_cat_short):
     ds = THREDDSMergedSource_cat.to_dask()
     ds_short = THREDDSMergedSource_cat_short.to_dask()
-    xr.testing.assert_equal(ds, ds_short)  # TODO: down load data only compare dims, coords, size
+    xr.testing.assert_equal(
+        ds, ds_short
+    )  # downloads data now, faster only compare dims, coords, size
```

Comment:
I didnt find assert_identical_coords_and_size
```suggestion
    for c in ds.coords:
        assert (ds[c] == ds_short[c]).all()
        assert ds.size == ds_short.size
```
I didnt find assert_identical_coords_and_size